src/tests/unit/tests/assessments/manual-test-record-your-results.test.tsx

```diff
@@ -3,28 +3,15 @@
 import * as Enzyme from 'enzyme';
 import * as React from 'react';
 
-import ManualTestRecordYourResults from '../../../../assessments/common/manual-test-record-your-results';
+import { ManualTestRecordYourResults } from '../../../../assessments/common/manual-test-record-your-results';
 
 describe('ManualTestRecordYourResultsTest', () => {
-    test('render: isMultipleFailurePossible = false', () => {
-        const props = { isMultipleFailurePossible: false };
-        const wrapper = Enzyme.shallow(<ManualTestRecordYourResults {...props} />);
-        const li = wrapper
-            .find('li')
-            .at(1)
-            .getElement();
-        expect(li.props.children[1]).toBe('a failure');
-        expect(li.props.children[5]).toBe(' then add the failure instance');
-    });
+    const multiple = [false, true];
 
-    test('render: isMultipleFailurePossible = true', () => {
-        const props = { isMultipleFailurePossible: true };
+    it.each(multiple)('- render: isMultipleFailurePossible = %p', isMultipleFailurePossible => {
```

Comment:
>-  [](start = 23, length = 2)

nit: why the extra punctuation?

Reply:
do you mean the '-'? forgot to remove it, it was so the "new" test didn't have the same name as the old ones... I'll update.